### DISCUSSION TITLE
Maybe add some contributors?

### DISCUSSION THREAD

**Discussion by Balssh** [prasanthrangan/hyprdots#495] [General]
As this project keeps growing and more people find out, maybe it would be a good idea to add a few people to help you @prasanthrangan with things, at least the "paperwork" like closing issues/reviewing PRs etc. I, for one, would gladly lend a hand.

**Comment by prasanthrangan** [comment]
Great, thanks for your support, I've added you as collaborator!
Please feel free to review and approve PR, but please don't merge it as some changes can break some scripts or config!

**Reply by Balssh** [reply]
Thanks, I'll probably mostly look through issues and close when one is solved. Currently I'm trying kde plasma so I won't have a close touch with the scripts.
